error.go

```diff
@@ -5,6 +5,7 @@ import (
 	"strings"
 )
 
+// Deprecated: will use Passage Error instead
```

Comment:
this deprecation comment could add a little more context.

if i were a dev reading this i don't think i'd quite understand what is going to happen.

including something like "Deprecated: this will be replaced by [PassageError] in v2" might make it easier to understand

Reply:
made this change